doc/classes/Vector4i.xml

```diff
@@ -194,6 +194,12 @@
 			<description>
 			</description>
 		</operator>
+		<operator name="operator []">
+			<return type="int" />
+			<argument index="0" name="index" type="int" />
+			<description>
```

Comment:
```suggestion
			<description>
				Access vector components using their index. [code]v[0][/code] is equivalent to [code]v.x[/code], [code]v[1][/code] is equivalent to [code]v.y[/code], [code]v[2][/code] is equivalent to [code]v.z[/code], and [code]v[3][/code] is equivalent to [code]v.w[/code].
```

Reply:
I wanted to leave Vector4i for another PR.